python/ray/train/BUILD

```diff
@@ -190,6 +190,14 @@ py_test(
     deps = [":train_lib"]
 )
 
+py_test(
+    name = "test_gpu_with_amp",
+    size = "large",
+    srcs = ["tests/test_gpu_with_amp.py"],
+    tags = ["team:ml", "exclusive", "gpu_only"],
+    deps = [":train_lib"]
+)
+
 py_test(
     name = "test_torch_fsdp",
     size = "large",
```

Comment:
need to add test_gpu_auto_transfer here too

Reply:
it's already there with `gpu_only` tag above

Reply:
ah sorry didnt see that!